subprojects/test-runner/device-provider/impl/src/main/kotlin/com/avito/android/runner/devices/internal/kubernetes/KubernetesReservationReleaser.kt

```diff
@@ -38,6 +38,19 @@ internal class KubernetesReservationReleaser(
         }
     }
 
+    suspend fun releaseDeployment(
+        deploymentName: String
```

Comment:
This deployment should be removed from `deployments` that we iterate through in `release`